doc/plotting.rst

```diff
@@ -86,6 +89,12 @@ xarray uses the coordinate name along with any associated `CF-compliant metadata
     @savefig plotting_1d_simple.png width=4in
     air1d.plot()
 
+xarray uses the coordinate name along with  metadata ``attrs.long_name``, ``attrs.standard_name``, ``DataArray.name`` and ``attrs.units`` (if available) to label the axes. The names ``long_name``, ``standard_name`` and ``units`` are defined by the `CF-conventions <http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch03s03.html>`_. When choosing names, the order of precedence is ``long_name``, ``standard_name`` and finally ``DataArray.name``. The y-axis label in the above plot was constructed from the ``long_name`` and ``units`` attributes of ``air1d``.
```

Comment:
Maybe same "copied from" instead of "defined by"?

Reply:
done.